dev-libs/quazip/patches/quazip-0.8.1.patchset

```diff
@@ -44,3 +44,73 @@ index abcdcf7..3355b61 100644
 -- 
 2.21.0
 
+
+From 163358c80f825c9954386f8ae02db3f3a18d83dd Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 6 Jul 2019 08:28:35 +0200
+Subject: Add pkgconfig support to cmake
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ddd681a..00d812c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -57,6 +57,7 @@ set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR})
+ 
+ set(LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)")
+ set(LIB_DESTINATION "${CMAKE_INSTALL_LIBDIR}" CACHE STRING "Library directory name" FORCE)
++set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_LIBDIR}/pkgconfig" CACHE STRING "Installation directory for pkgconfig (.pc) files" FORCE)
+ set(QUAZIP_LIB_TARGET_NAME quazip${QUAZIP_LIB_VERSION_SUFFIX} CACHE
+     INTERNAL "Target name of libquazip" FORCE)
+ 
+diff --git a/quazip.pc.cmakein b/quazip.pc.cmakein
+new file mode 100644
+index 0000000..d342568
+--- /dev/null
++++ b/quazip.pc.cmakein
+@@ -0,0 +1,12 @@
++prefix=@CMAKE_INSTALL_PREFIX@
++exec_prefix=${prefix}
++libdir=@CMAKE_INSTALL_LIBDIR@
++includedir=@CMAKE_INSTALL_INCLUDEDIR@/quazip@QUAZIP_LIB_VERSION_SUFFIX@
++
++
++Name: Quazip
++Description: Quazip Library
++Version: @QUAZIP_LIB_VERSION@
++Libs: -L${libdir} -lquazip5
```

Comment:
Need to check this in a bit, but think you can use @QUAZIP_LIB_VERSION_SUFFIX@ also for the "5" :)